kvdb-rocksdb/Cargo.toml

```diff
@@ -27,6 +27,7 @@ parity-util-mem = { path = "../parity-util-mem", version = "0.4", default-featur
 [dev-dependencies]
 alloc_counter = "0.0.4"
 criterion = "0.3"
+ethereum-types = { path = "../ethereum-types" }
```

Comment:
Why was this needed?

Reply:
I broke this in https://github.com/paritytech/parity-common/pull/301/files#diff-3206254ae7bfdf4d978075eb4cafbdcfR30, but we didn't have CI checks to catch it because `keccak-hash` benches didn't compile on stable I think?

Reply:
(this is a bench-only dependency)